src/components/DateTimePicker.vue

```
@@ -0,0 +1,202 @@
+
+<script setup lang="ts">
+import { ref, computed, watch } from "vue";
+import moment from "moment";
+import { useI18n } from "vue-i18n";
+
+const props = defineProps<{
+  modelValue: Date | null;
+  placeholder: string;
+  minDate?: Date;
+  maxDate?: Date;
+  disabledDates?: Date[];
```

ESLint

[error] 'disabledDates' of property found, but never used.

(vue/no-unused-properties)